vm/actor/src/builtin/power/policy.rs

```diff
@@ -1,107 +1,66 @@
 // Copyright 2020 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use super::{SectorStorageWeightDesc, SectorTermination};
-use crate::{reward, StoragePower};
-use clock::ChainEpoch;
-use num_bigint::{BigInt, BigUint};
-use num_traits::{FromPrimitive, Pow};
-use runtime::ConsensusFaultType;
+use super::SectorStorageWeightDesc;
+use fil_types::{SectorQuality, StoragePower};
+use num_bigint::BigInt;
+use num_traits::{FromPrimitive, ToPrimitive};
 use vm::TokenAmount;
 
-/// The time a miner has to respond to a surprise PoSt challenge.
-pub const WINDOWED_POST_CHALLENGE_DURATION: ChainEpoch = 240; // ~2 hours @ 30 second epochs. PARAM_FINISH
-
-/// The number of consecutive failures to meet a surprise PoSt challenge before a miner is terminated.
-pub const WINDOWED_POST_FAILURE_LIMIT: i64 = 3; // PARAM_FINISH
-
-/// Minimum number of registered miners for the minimum miner size limit to effectively limit consensus power.
-pub const CONSENSUS_MINER_MIN_MINERS: usize = 3;
-
-/// Maximum age of a block header used as proof of a consensus fault to appear in the chain.
-pub const CONSENSUS_FAULT_REPORTING_WINDOW: ChainEpoch = 2880; // 1 day @ 30 second epochs.
-
 lazy_static! {
-    /// Multiplier on sector pledge requirement.
-    pub static ref PLEDGE_FACTOR: BigUint = BigUint::from(3u8); // PARAM_FINISH
-
-    /// Total expected block reward per epoch (per-winner reward * expected winners), as input to pledge requirement.
-    pub static ref EPOCH_TOTAL_EXPECTED_REWARD: BigUint = reward::BLOCK_REWARD_TARGET.clone() * 5u8; // PARAM_FINISH
-
     /// Minimum power of an individual miner to meet the threshold for leader election.
     pub static ref CONSENSUS_MINER_MIN_POWER: StoragePower = StoragePower::from_i32(2 << 30).unwrap(); // placeholder
 
+    pub static ref BASE_MULTIPLIER: BigInt = BigInt::from(10); // PARAM_FINISH
+    pub static ref DEAL_WEIGHT_MULTIPLIER: BigInt = BigInt::from(11); // PARAM_FINISH
+    pub static ref VERIFIED_DEAL_WEIGHT_MULITPLIER: BigInt = BigInt::from(100); // PARAM_FINISH
+    pub static ref SECTOR_QUALITY_PRECISION: BigInt =  BigInt::from(20); // PARAM_FINISH
```

Comment:
These probably don't need to be big ints, the reason the go impl might have them as such is because they can only do arithmetic on big ints with other big ints, we have the luxury of doing the arithmetic with other types (but of course verify the usage)